jmclient/jmclient/maker.py

```diff
@@ -254,6 +255,23 @@ def modify_orders(self, to_cancel, to_announce):
                     self.offerlist.remove(oldorder_s[0])
             self.offerlist += to_announce
 
+    def freeze_timelocked_utxos(self):
+        """
+        Freeze all wallet's timelocked UTXOs. These cannot be spent in a
+        coinjoin because of protocol limitations.
+        """
+        if not hasattr(self.wallet_service.wallet, 'FIDELITY_BOND_MIXDEPTH'):
```

Comment:
is `isinstance` not viable here? I thought that worked OK for mixins?

Reply:
`isinstance` is not pythonic. It violates the duck-typing philosophy ("If it quacks like a duck it is a duck."). It makes sense for base types (that don't have identifying attributes) but usually not for more complex objects.